api/samples/cbrtrace.c

```diff
@@ -75,10 +80,10 @@ event_thread_init(void *drcontext)
 {
     file_t log;
     log =
-        log_file_open(client_id, drcontext, NULL /* using client lib path */, "cbrtrace",
+            log_file_open(client_id, drcontext, NULL /* using client lib path */, "cbrtrace",
```

Comment:
Please maintain the clang-format style.  This is likely why all the tests are failing.